packages/components/src/FormField/FormField.css

```diff
@@ -215,10 +215,16 @@
   flex-direction: column;
 }
 
+/* If there's a label, add padding-top to wrapper to account for "absolute" positioned label */
```

Comment:
this is largely a personal preference so I could be convinced otherwise but I usually want a fairly compelling reason to add comments

I'd say the code communicates the first half of this comment - if there's a label, add padding

so it's really only the "why" around the absolutely positioned label that the comment adds. I guess my question would be, did you find that part unintuitive or hard to understand to the point where you think a comment explaining the padding would save time or effort?